en/7-iterate-improve-frequently.md

```diff
@@ -32,390 +15,170 @@ altLangPage: 7-effectuer-iterations-ameliorations-constamment
 
 <section class="dpgn-section-guideline">
 
-## 7.1 Build in an agile manner and continuously improve in response to user needs
+## 7.1  Build in an agile manner and continuously improve in response to user needs
 
 <div class="dpgn-section-intro-guideline">
 
-**[TODO: Add/revise introductory text]**
-
-**Introduction examples:**
-
-**Digital Service Standard (UK):** Build your service using the agile, iterative and user-centred methods set out in the manual.
-
-Using agile methods helps you to build services that:
-
-- meet the needs of your users
-- are easy and convenient for people to use
-- you can change easily, eg if government policy or technology policy changes
-- you can keep improving, eg based on user feedback
-- cost less and are more accountable
-
-**Digital Service Standard (Ontario):** Design and build the service using an agile and user-centred approach. Agile is an approach to building services that breaks the work into smaller chunks known as iterations. Build one feature of the service at a time until the entire service is complete.
-
-It is a much lower risk approach than traditional build-it-all-at-once approach known as waterfall because frequent iterations expose any flaws in the original plan much faster (e.g. not getting approvals, not enough resources, not the right people on the team, etc.)
-
-User-centred methods such as user research and usability testing put the focus on making services that are easy-to-use. Traditional government services focus on meeting business needs and aligning with policy goals. A user-centred approach ensures business needs are also balanced against user needs. This helps to increase digital service uptake.
+Design and build services using an agile and user-centred approach. Agile is an approach to building services that breaks the work into smaller chunks known as iterations. Build one feature of the service at a time until the entire service is complete.  It is a much lower risk approach than traditional build-it-all-at-once approach known as waterfall because frequent iterations expose any flaws in the original plan much faster (e.g. not getting approvals, not enough resources, not the right people on the team, etc.) **(Digital Service Standard (Ontario))**
```

Comment:
Is this new content sourced from the Ontario Digital Service Standard? If not, you should remove the reference.